File: androidApp/src/main/java/com/medhelp/medhelp/data/model/BonusesResponse.java
package com.medhelp.medhelp.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class BonusesResponse {
    @SerializedName("error")
    private boolean error;

    @SerializedName("message")
    private String message;

    @SerializedName("response")
    private List<BonusesItem> response;

    public boolean isError() {
        return error;
    }

    public void setError(boolean error) {
        this.error = error;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<BonusesItem> getResponse() {
        return response;
    }

    public void setResponse(List<BonusesItem> response) {
        this.response = response;
    }
}

File: shared/src/commonMain/kotlin/com/medhelp/newmedhelp/model/BonusesResponse.kt
package com.medhelp.newmedhelp.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable


@Serializable
class BonusesResponse {
    @SerialName("error")
    var error = false

    @SerialName("message")
    var message: String? = null

    @SerialName("response")
    var response: List<BonusesItem>? = null
}